pages/linux/whatis.md

```diff
@@ -10,6 +10,10 @@
 
 `whatis --long {{command}}`
 
+- Display descriptions for all commands matching a glob:
+
+`whatis --wildcard {{command_glob}}`
```

Comment:
Maybe we should differentiate a bit from glob and regex. net* or tcp* as a glob is good. For regex, can use "wish[0-9].[0-9]".